src/main/scala/ch/epfl/bluebrain/nexus/iam/acls/AclEvent.scala

```diff
@@ -110,17 +111,17 @@ object AclEvent {
   object JsonLd {
     import io.circe.generic.extras.semiauto._
 
-    private implicit val config: Configuration = Configuration.default
-      .withDiscriminator("@type")
-      .copy(transformMemberNames = {
-        case "rev"     => "_rev"
-        case "instant" => "_instant"
-        case "subject" => "_subject"
-        case "path"    => "_path"
-        case other     => other
-      })
-
+    @silent // defined implicits are not recognized as being used
```

Comment:
There is no need to build the configuration every time inside the `def` right?

```
private[acls] implicit val config: Configuration = Configuration.default
        .withDiscriminator("@type")
        .copy(transformMemberNames = {
          case "rev"     => "_rev"
          case "instant" => "_instant"
          case "subject" => "_subject"
          case "path"    => "_path"
          case other     => other
        })
```

Reply:
Technically no, but this way it's easier to understand where the config is used + we use a single `@silent`.